### DISCUSSION TITLE
An experiment using Dreambooth

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#2835] [Q&A]
After the self-destruction of Kohya, I don't have a way to finetune a model using colab. Dreambooth is great for single subjects, so I was wondering what would happen if you train a vast amount of images with text captions each one with a different filename. I don't know if Ben is gonna read this, but what do you think? Let's say I train 1000 images with random names but with text captions.

**Comment by TheLastBen** [comment]
you can either use the filenames as the captions (use _ for spaces), or use text files named the same as each corresponding image
